microsoft/PowerToys
# Move hardcoded strings in Keyboard Manager projects to rc file

## Summary of the Pull Request

_What is this about?_

This PR moves all the hardcoded strings in the Keyboard Manager projects to a .rc file.

## PR Checklist
* [X] Applies to #1880 

## Info on Pull Request

_What does this include?_
- Moved strings for UI elements to rc file
- Moved strings for error warnings to rc file
- Moved strings for window title and window creation/class registration error messages to rc file

## Pending Work for the issue
The key names for special keys (non unicode charkeys) are currently hardcoded in the common project at https://github.com/microsoft/PowerToys/blob/master/src/common/keyboard_layout.cpp#L104-L217. These should be localized as well after identifying the best practice to localize strings in common.

## Validation Steps Performed

_How does someone test & validate?_
Open Remap keys and remap shortcuts to ensure all strings appear.
